### ISSUE TITLE
Allow explicit restarting from RAD files (or other points?)

### ISSUE THREAD

**Issue by jashapiro** [AlexsLemonade/scpca-nf#41]
While the `-resume` option in nextflow is great for when a workflow is repeated shortly after running and/or failures, most intermediate files are not stored long-term, so a later rerun with changes in parameters or other code could well result in rerunning the full workflow. 

Since we are storing intermediate files (in particular RAD files), it might be nice to have an option in the workflow that would skip mapping (the most time-consuming step!) if the RAD files are available (after #40 is implemented).

I am not exactly sure what the implementation of this would look like at this stage... We will have to do some investigation of the best way to test for the presence of an alevin output directory and run a different section of the workflow in that case.
